recipes/mrtrix3/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mrtrix3" %}
+{% set version = "3.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MRtrix3/mrtrix3/archive/{{ version }}.tar.gz
+  sha256: 6ec7d5a567d8d7338e85575a74565189a26ec8971cbe8fb24a49befbc446542e
+  patches:
+    - 0001-remove-Eigen-Vector3-4-typedefs.patch
+    - 0002-minor-changes-to-compile-with-Eigen-3.4.patch
+    - 0003-tests-reduce-tolerance-on-failing-tests.patch
+    - 0004-No-compile-test-for-dependencies.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
+    - pkg-config
+  host:
+    - eigen
+    - fftw
+    - libpng
+    - libtiff
+    - zlib
+
+test:
+  commands:
+    - mrconvert
```

Comment:
Looks like the python module is installed as part of this command:
```bash
cp -r bin lib share "$PREFIX"
```
My suggestion is to start by making a `setup.cfg`, `pyproject.toml` and installing the python module using pip instead of direct copy. These files can be added to the recipe folder and then moved into the upstream source at a later date.